impl/sql/mutationstorage/mutations_test.go

```diff
@@ -246,7 +230,8 @@ func TestReadAll(t *testing.T) {
 			},
 		},
 	} {
-		maxSequence, results, err := readAll(ctx, m, tc.startSequence)
+		maxcount := 1000
```

Comment:
Nit: `maxCount`. Do you want this to be `const`?

Perhaps it ought to be a field of the test table so you can test with boundary and invalid values?

Reply:
added